models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-germany.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { TZoneDraft, TZoneDraftGraphql } from '../../../types';
+import countryGermany from './country-germany;
```

Comment:
You're missing an end quote!

```suggestion
import countryGermany from './country-germany';
```